src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/alert-title-template.mdx

```diff
@@ -1,67 +1,87 @@
 ---
-title: Title template for NRQL Alert Conditions
+title: Create a title for a NRQL alert condition
```

Comment:
This isn't accurate. It's not a title for a NRQL alert condition. It's a title _template_ for a NRQL alert condition which becomes the title when an incident is opened.

Reply:
I fixed this!